platform/state/platform_state.proto

```diff
@@ -195,3 +211,134 @@ message MinimumJudgeInfo {
      */
     uint64 minimum_judge_ancient_threshold = 2;
 }
+
+/**
+ * A network address book.<br/>
+ * The address book records the address of every known consensus node that
+ * participates in the network, including `0 weight` nodes.<br/>
+ */
+message AddressBook {
+  /**
+   * A consensus round.<br/>
+   * The round when this address book was created.
+   */
+  int64 round = 1;
+
+  /**
+   * The node ID of the next address that can be added.
+   * <p>
+   * The next node identifier assigned SHALL be equal to this value.<br/>
+   * All existing node identifiers SHALL be strictly less than this value.<br/>
+   */
+  NodeId next_node_id = 2;
```

Comment:
Is this needed?  I assume we're setting the address book information from `config.txt` rather than computing node identifiers separately.